redhat-acm/authorization/disallowed-role-policy/README.md

```diff
@@ -0,0 +1,3 @@
+# Red Hat Advanced Cluster Management for Kubernetes Authrorization Policies
+
+This directory includes a list of Red Hat Advanced Cluster Management for Kubernetes policies that focus on authorization in OpenShift.
```

Comment:
This file should be located under redhat-acm/authorization and not redhat-acm/authorization/disallowed-role-policy

furthermore, I see you generated a copy of it in all of the README.md in the sub-directories under authorization - which is unnecessary duplication